static/styles/subscriptions.css

```diff
@@ -320,16 +304,16 @@ h4.stream_setting_subsection_title {
             color: inherit;
         }
     }
+
+    .form-inline {
+        margin-bottom: 0;
+    }
 }
 
 .subscriber-search {
     margin: 10px 0 0;
 }
 
-.subscriber_list_add .form-inline {
-    margin-bottom: 0;
-}
-
```

Comment:
Added this `.form-inline` margin-bottom inside the nested CSS for `.subscriber_list_add`